README.md

```diff
@@ -4,3 +4,72 @@
 [![codecov](https://codecov.io/gh/OpenSource-THG/cfssl-issuer/branch/master/graph/badge.svg)](https://codecov.io/gh/OpenSource-THG/cfssl-issuer)
 [![Go Report Card](https://goreportcard.com/badge/github.com/Opensource-THG/cfssl-issuer)](https://goreportcard.com/report/github.com/Opensource-THG/cfssl-issuer)
 ![Docker Pulls](https://img.shields.io/docker/pulls/opensourcethg/cfssl-issuer)
+
+CFSSL Issuer is a controller that extends Jetstack's [cert-manager](https://github.com/jetstack/cert-manager) to add an issuer that uses a
+CFSSL server to sign certificate requests.
+
+## Installation
+
+This controller requires a cert-manager version of > v0.11.0 and a running cfssl server
+
+### Helm
+
+TBD
+
+### Manually
+
+```bash
+git clone git@github.com:OpenSource-THG/cfssl-issuer.git
+cd cfssl-issuer
+kubectl apply -f deploy
+```
+
+## Usage
+
+Once installed we need to configure either a CfsslIssuer or CfsslClusterIssuer resource.
+
+```yaml
+kind: CfsslIssuer
+apiVersion: certmanager.thg.io/v1beta1
+metadata:
+  name: cfsslissuer-server
+spec:
+  url: https://cfsslapi.local
+  caBundle: <base64-encoded-ca>
+  profile: server
```

Comment:
nit: Mention that `profile` is optional?

Reply:
I've expanded the whole section to be more explicit